autotest/gdrivers/grib.py

```diff
@@ -55,8 +55,9 @@ def has_jp2kdrv():
 
 def test_grib_1():
 
-    tst = gdaltest.GDALTest('GRIB', 'grib/ds.mint.bin', 2, 46927)
-    return tst.testOpen()
+    with gdaltest.config_option('GDAL_PAM_ENABLED', 'OFF'):
```

Comment:
we should solve the .aux.xml issue. Setting explicitly GDAL_PAM_ENABLED is not pretty